docs/migration_guide.rst

```diff
@@ -1,6 +1,27 @@
 Migration guide
 ===============
 
+**********************
+0.16.0 Migration guide
+**********************
+
+Deprecation
+-----------
+
+1. Deprecated Client's :meth:`~starknet_py.net.client.Client.net` property.
+
+Breaking changes
+----------------
+
+1. Removed deprecated ``AccountClient``
```

Comment:
I once got a comment from someone that we do not have to write `deprecated` everywhere. 
Maybe let's write which class should be used instead?